Tests/SonarScanner.MSBuild.TFS.Tests/BuildVNextCoverageReportProcessorTests.cs

```diff
@@ -359,5 +381,20 @@ public void ProcessCoverageReports_NotVsCoverageXmlPathProvided_NotCoverageXmlFi
             result.Should().BeTrue();
             converter.AssertExpectedNumberOfConversions(1);
         }
+
+        [ClassCleanup]
+        public static void AddFilesToDeleteToEnv() {
+
+            filesToDelete = filesToDelete.Distinct().ToList();
+
+            var currentEnvValue = Environment.GetEnvironmentVariable("TEST_FILE_TO_DELETE", EnvironmentVariableTarget.User) ?? String.Empty;
+            if (!string.IsNullOrEmpty(currentEnvValue))
+            {
+                currentEnvValue += ";;";
+            }
+
+            Environment.SetEnvironmentVariable("TEST_FILE_TO_DELETE", currentEnvValue + string.Join(";;",
```

Comment:
Why not just delete the files here?

Reply:
I tried, but they're locked at runtime by the vstest.exe process unfortunately.

Reply:
Ok.
It's not obvious why this method is needed. A comment would help, or maybe just change the name of the environment variables to something like `S4MSB_TEST_FILES_TO_DELETE_IN_AZURE_PIPELINE_YML`

Reply:
One other option - I don't think it matter to the unit tests where the file is written, so we just write them somewhere the Azure pipeline won't look e.g. in a sub-folder under %TEMP%? Then we could avoid all of the code to track and delete files.

Reply:
I'll try it, i remember trying with another TestResults folder name only, but that was not working since this name is hardcoded in the code.